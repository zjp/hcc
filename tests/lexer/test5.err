FATAL [1,1]: Integer literal too large;  using max value
FATAL [2,1]: Integer literal too large;  using max value
