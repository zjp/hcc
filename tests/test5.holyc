123423452345 
123452345234523452345234523452345345
