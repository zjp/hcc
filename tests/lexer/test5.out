INTLIT:2147483647 [1,1]
INTLIT:2147483647 [2,1]
EOF [3,1]
